test-js/__snapshots__/analyze.test.js.snap

```diff
@@ -0,0 +1,23 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`analyze in ./dsl matches snapshot 1`] = `
+{
+  "stderr": "
+~/Eask:7:18 Error: Multiple definition of \`package'
+~/Eask:10:55 Error: Multiple definition of \`website-url'
+~/Eask:12:16 Error: Multiple definition of \`keywords'
+~/Eask:15:15 Warning: Warning regarding duplicate author name, name
+~/Eask:18:17 Warning: Warning regarding duplicate license name, GPLv3
+~/Eask:21:29 Error: Multiple definition of \`package-file'
+~/Eask:23:35 Warning: Pkg-file seems to be missing \`check-pkg.el'
+~/Eask:24:35 Error: Multiple definition of \`package-descriptor'
+~/Eask:28:61 Error: Run-script with the same key name is not allowed: \`test\`
+~/Eask:31:13 Error: Multiple definition of source \`gnu'
+~/Eask:33:24 Error: Unknown package archive \`magic-archive'
+~/Eask:36:20 Error: Define dependencies with the same name \`emacs'
+~/Eask:39:19 Error: Define dependencies with the same name \`dash'
+~/Eask:44:2 Error: Define dependencies with the same name \`f'
+",
+  "stdout": "",
+}
+`;
```

Comment:
Should `__snapshots__` be ignored?

I've googled it, and it seems like we should commit to the repo, but I would still like to double-check! 😁

Reply:
Yep that's right, it should be committed. I've added some methods that help filter out local data in snapshots, it's in the new doco file.